### PR TITLE
Feature/go synchronizer

### PR DESCRIPTION
**Pull request by kaiden-araki**
## Merge `feature/go-synchronzier` branch into `develop` branch

- All the updates for synchronizer since last sync with `feature/go-sequencer` branch.
- Resolve merge conflicts.

### CODE REVIEW

File: sequencer/.env [hunk 1]
```diff
@@ -0,0 +1,5 @@
+BootCoordinator=
+BootCoordinatorURL=
+PGUSER=postgres
+PGPASSWORD=123
+PGDATABASE=statedb
```
Comment (Intizar-T) [inline sequencer/.env:5]: I think we should remove `.env` from prod. I added it to `.gitignore` and created `.env.example`. Also mentioned to `cp .env.example .env` in the sequencer readme

**Review by Intizar-T — DISMISSED** [review]
LGTM!

File: sequencer/.env (hunk 1 above)
Comment (kaiden-araki) [reply sequencer/.env]: Critical comments.
I have removed .env file.

**Review by Shailu-s — DISMISSED** [review]

**Review by Intizar-T — APPROVED** [review]
lgtm!

**Pull request merged**